composer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "alps/gutenberg",
+  "type": "wordpress-plugin",
+  "version": "1.5.1",
```

Comment:
Will we need to update this on every new build? Can we pull this from the `kernl.version` file as part of a `npm build` script?